public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -135,21 +135,21 @@ block get-heroes-details
 
     For *now* we get back on familiar ground by immediately by 
     converting that `Observable` to a `Promise` using the `toPromise` operator.
-  +makeExample('toh-6/ts/app/hero.service.ts', 'to-promise')(format=".")
+  +makeExcerpt('app/hero.service.ts', 'to-promise')
```

Comment:
Oups, didn't mean for that to happen. I forgot that the convention for including/excluding a title with `makeExcerpt` is reversed as compared to `makeExample`. Fixed all of excerpts that originally did not have a title.